src/suites/cts/command_buffer/compute/sampled_texture_clear.spec.ts

```diff
@@ -0,0 +1,75 @@
+export const description = `
+computePass test that sampled texture is cleared`;
+
+import { TestGroup } from '../../../../framework/index.js';
+import GLSL from '../../../../tools/glsl.macro.js';
+import { GPUTest } from '../../gpu_test.js';
+
+export const g = new TestGroup(GPUTest);
+
+g.test('compute pass test that sampled texture is cleared', async t => {
+  const texture = t.device.createTexture({
+    size: { width: 1, height: 1, depth: 1 },
```

Comment:
Without too much trouble, I think we could make this more robust by testing a larger texture. You can just do a similar copy from texture into buffer, but with a larger texture, larger dispatch, and larger buffer, then check the whole buffer at the end. I'd make it pretty large, say 256x256.